datalakehouse-api/lakehouse_query_api.py

```
@@ -0,0 +1,931 @@
+            "code": 200,
+            "table": request.table_name,
+            "row_count": len(data),
+            "data": data
+        }
+    except HTTPException:
+        raise
+    except ValueError as ve:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail={"status": "error", "code": 404, "message": str(ve)[:120], "error_type": "ValueError"}
+        )
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)